ffi-ctests/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+#[test]
+fn test() {
```

Comment:
That bothers me that we need separate crate just for tests and all that we do on the Rust side is just check boolean results of the functions. I suggest to take some minimal C unit testing framework (e.g. https://github.com/h2o/picotest) and just add tests in the `ffi` crate at test purely on the C side.

Reply:
I think current approach is fine, but it would be nice to at least move this crate inside of `test` folder of `wirefilter-ffi` to make it clear that it's used just for testing and to avoid building tests as part of the regular build.

Reply:
It now uses the `cc` crate and I also moved the `wirefilter-ffi-ctests` crate in `ffi/tests/ctests` and made `wirefilter-ffi` dev-depends on it.